src/app/main/component/user/components/profile/users-friends/friend-dashboard/friend-dashboard.component.html

```diff
@@ -16,7 +16,7 @@ <h1>{{ 'profile.friends.friends' | translate }}</h1>
             type="text"
             #search
             [placeholder]="'search.search-popup.placeholder' | translate"
-            (input)="onInput(search)"
+            (keyup)="onKeyup(search)"
```

Comment:
Why not `input`? Because with key up with input will be triggered on any keys, not only on change the value

Reply:
It's strange because the (input) event in Angular is used to handle the input events on an input element. It triggers every time the value of the input element changes.